internal/worker/rundynamic.go

```diff
@@ -50,9 +53,20 @@ func RunDynamicAnalysis(pkg *pkgecosystem.Pkg, sbOpts []sandbox.Option) (result.
 	var lastStatus analysis.Status
 	var lastError error
 	for _, phase := range pkg.Manager().RunPhases() {
-		result, err := dynamicanalysis.Run(sb, pkg.Command(phase))
+		startTime := time.Now()
+		phaseResult, err := dynamicanalysis.Run(sb, pkg.Command(phase))
 		lastRunPhase = phase
 
+		runDuration := time.Since(startTime)
+		log.Info("Dynamic analysis phase finished",
+			"ecosystem", pkg.EcosystemName(),
```

Comment:
`ecosystem` and `phase` and `status` can all be wrapped in `log.Label()` as they have fixed cardinality.